ssb-broadcast.sh

```diff
@@ -4,11 +4,11 @@ while true; do
     id=$(sbot whoami | grep id | awk -F "\"" '{print $4}' | sed 's/.ed25519//' | sed 's/@//')
     if ! [ -z "$id" ]; then
         for int in $(ls -1Atu /sys/class/net); do
-            ip=$(ip addr show $int | grep -v inet6 | grep -v '127.0.0.1' |grep inet | head -n 1 | awk '{print $2}' | awk -F "/" '{print $1}')
+            ip=$(ip addr show $int | grep -v inet6 | grep -v '127.0.0.1' | grep inet | head -n 1 | awk '{print $2}' | awk -F "/" '{print $1}')
```

Comment:
does `ip=$(ip addr show $int | grep -oE '[0-9.]+/16)` do the same thing here?

Reply:
or `ip=$(ip addr show $int | grep -oE '([0-9]{1,3}\.){3}[0-9]{1,3}'` if you want a bit more sane restrictions

Reply:
Then you get broadcast addresses to

```
pi@raspberrypi:~ $ ip addr show $int | grep -oE '([0-9]{1,3}\.){3}[0-9]{1,3}'
192.168.1.211
192.168.1.255
```

Reply:
I will keep it as is for now then